appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: "{build}"
+
+environment:
+  nodejs_version: "6"
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - npm install
+
+test_script:
+  - node --version
+  - npm --version
+  - npm test
+
+build: off
+
+branches:
+
+  only:

```

Comment:
are you sure there should be line breaks before and after this? YAML usually cares about whitespaces.